src/test/rustdoc-ui/deny-intra-link-resolution-failure.stderr

```diff
@@ -0,0 +1,13 @@
+error: `[v2]` cannot be resolved, ignoring it...
+  --> $DIR/deny-intra-link-resolution-failure.rs:13:6
+   |
+13 | /// [v2] //~ ERROR
+   |      ^^ cannot be resolved, ignoring
+   |
+note: lint level defined here
+  --> $DIR/deny-intra-link-resolution-failure.rs:11:9
+   |
+11 | #![deny(intra_doc_link_resolution_failure)]
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   = help: to escape `[` and `]` characters, just add '/' before them like `/[` or `/]`
```

Comment:
Is this really how it comes out when the error appears? Or is it an artifact of the UI testing? It would be really worrying if we were accidentally suggesting the wrong thing.

Reply:
@QuietMisdreavus Just an artifact of UI test normalization.

Reply:
Phew, i was getting nervous there. Imperio said he got the proper output when he ran the test locally, too, so i'll let this slide.